bridge-history-api/internal/utils/utils.go

```diff
@@ -172,13 +172,16 @@ func GetBlocksInRange(ctx context.Context, cli *ethclient.Client, start, end uin
 
 // ConvertBigIntArrayToString convert the big int array to string
 func ConvertBigIntArrayToString(array []*big.Int) string {
```

Comment:
can you add unit test for your changes?

Reply:
I only modified the internal structure of the function and ran the `TestConvertBigIntArrayToString` test. The result passed successfully.

Reply:
I only modified the internal structure of the function and ran the TestConvertBigIntArrayToString test. The result passed successfully.